detection-rules/impersonation_fake_msg_thread_mismatched_from_freemail_replyto.yml

```
@@ -0,0 +1,42 @@
+name: "Fake Message Thread - First Time Sender with a Mismatched Freemail Reply-To Address"
+description: |
+  Fake Message Threads or Chain Reuse is a common confidence technique exploited by threat actors to bolster credibility.
+  This is typically used in conjunction with a reply-to address that is not the same as the sender address. 
+
+references: 
+  - "https://playground.sublimesecurity.com?id=5ab9a225-4de3-494f-9a55-e16ec9e1e5c3"
+type: "rule"
+severity: "medium"
+source: |
+   type.inbound
+   // First Time Sender
+   and (
+           (
+               sender.email.domain.root_domain in $free_email_providers
+               and sender.email.email not in $sender_emails
+       or (
+               sender.email.domain.root_domain not in $free_email_providers
+               and sender.email.domain.domain not in $sender_domains
+           )
+       ) 
+   and any(headers.reply_to, .email.domain.domain in $free_email_providers
+   and not .email.domain.domain == sender.email.domain.domain)
+   and not strings.ilike(sender.email.local_part, "support", "sales", "noreply", "marketing")
+   
+   // Check for Message Thread Indicators
+   and (
+       strings.istarts_with(subject.subject, "RE:")
+   or any(
+       [body.plain.raw, body.html.display_text], regex.icontains(.,"From:[ a-z0-9<>_@\\.]{0,80}Sent:[ a-z0-9<>_@\\.:]{0,40}To:[ a-z0-9<>_@\\.;]{0,300}(Cc:)?.{0,300}Subject:"))
+   
+       )
+   // Check for the Presence of References or In-Reply-To properties
+   and (
+       length(headers.references) == 0
+   or not any(
+       headers.hops, any(.fields, strings.ilike(.name, "In-Reply-To")))
+       )
+   )
+tags:
+  - "Contact Impersonation"
```

```suggestion
  - "Contact impersonation"
```
for consistency with existing rules. Tags are case-sensitive (will show up twice in the `Tags` list)